packages/clay-form/src/index.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import Input, {IProps as IInputProps} from './Input';
+
+const ClayForm: React.FunctionComponent<
```

Comment:
Rather than rolling out our own Form component and other elements inside of the form, what do you think about just using `formik`? I believe the personalization and segments editor in portal is already using it.

Reply:
Anecdote: We rolled our own form component and elements for both Loop and Faro, and it was one of the hardest and most annoying areas of the app we had to maintain and fix issues in.

Reply:
I agree with you, it's too bad to deal with Forms, the idea of `<ClayForm />` is to just provide a few markups with a few features and it's easy to integrate with `formik`, the part I have not yet started exploring.

Maybe it's something to explore in upcoming releases or add along with these new features.

Reply:
Yeah I think it might be best to keep our Form and form elements as simple as possibly where we simply add the appropriate classes, type requirements, and then pass everythign else in via `...otherProps` unless there is a functionality difference with clay. This gives us freedom to default to the user using their own callbacks or formik.

Reply:
Yeah, I think we can then add a use case of Formik with ClayForm in Storybook.